packages/@aws-cdk/aws-rds/lib/cluster.ts

```diff
@@ -684,7 +692,7 @@ interface InstanceConfig {
  * A function rather than a protected method on ``DatabaseClusterNew`` to avoid exposing
  * ``DatabaseClusterNew`` and ``DatabaseClusterBaseProps`` in the API.
  */
-function createInstances(cluster: DatabaseClusterNew, props: DatabaseClusterBaseProps, subnetGroup: CfnDBSubnetGroup): InstanceConfig {
+function createInstances(cluster: DatabaseClusterNew, props: DatabaseClusterBaseProps, subnetGroupName: string): InstanceConfig {
```

Comment:
Can we actually make this function take an `ISubnetGroup` as argument instead of `string`? I believe this would entail creating a`this.subnetGroup` of type `ISubnetGroup`, and getting rid of `this.subnetGroupName`.